src/bowlingGame.js

```diff
@@ -0,0 +1,99 @@
+var BowlingGame = function() {
+
+  this.scoreGrid = {1: null,
+                      2: null,
+                      3: null,
+                      4: null,
+                      5: null,
+                      6: null,
+                      7: null,
+                      8: null,
+                      9: null,
+                      10: null,
+                    };
+
+  this.currentGame = {1: [null,null],
+                  2: [null,null],
+                  3: [null,null],
+                  4: [null,null],
+                  5: [null,null],
+                  6: [null,null],
+                  7: [null,null],
+                  8: [null,null],
+                  9: [null,null],
+                  10: [null,null,null],
+                };
+
+  this.score = 0;
+
+};
+
+BowlingGame.prototype.currentScore = function() {
+  currentGame = this.currentGame;
+  scoreGrid = this.scoreGrid;
+  for (i = 1; i <= 9; i++) {
+    nextFrame = i + 1;
+    nextNextFram = i + 2;
+    nextNextFrame = i + 2;
+    if (game.isNotSpareOrStrike(i) && game.currentGame[i][1] != null) {
```

Comment:
All these ifs are really hard to understand. Try to extract out some of this logic to other descriptively-named functions.